magento/magento2
# magento/magento2#30408: solved multishipping cart item subtotal updat…

### Description (*)

Multishipping does not update cart item subtotal correctly
### Related Pull Requests


### Fixed Issues (if relevant)

1. Fixes magento/magento2#30408

### Manual testing scenarios (*)


1. Add product in the cart
2. Click on Check Out with Multiple Addresses Link
3. Back to the cart page
4. Update product qty and click on Update Shopping Cart

### Questions or comments


### Contribution checklist (*)
